src/lib/app-switcher/app-switcher-menu/app-switcher-menu.component.html

```diff
@@ -0,0 +1,34 @@
+<td-menu class="app-switcher-menu">
+  <!--header-->
+  <div td-menu-header class="app-switcher-header">
+    <mat-icon
+      class="logo-icon"
+      *ngIf="_themeService.darkTheme$ | async; else lightLogo"
+      svgIcon="td-logo:on-dark"
```

Comment:
Instead  of having an else, we can just use the theme map method on the svgIcon

Reply:
Yep, figured out how to do this in #80

Reply:
That makes sense. Let me try that out

Reply:
Changed~

Reply:
oh wait, do we need to change how we use the logo @adamnel  with that other PR?

Reply:
I don't think it will break it but it's a more self-contained way to do this. Maybe a separate PR?

Reply:
Sounds good